### ISSUE TITLE
download_jar fails at following redirect

### ISSUE THREAD

**Issue by hennr** [jamesdbloom/mockserver-node#12]
Running ```npm install && npm run test``` on my system fails with the following error:

```
Fetching {"host":"oss.sonatype.org","path":"/content/repositories/releases/org/mock-server/mockserver-netty/5.5.1/mockserver-netty-5.5.1-jar-with-dependencies.jar","port":443,"followAllRedirects":true}
Fetching {"host":"oss.sonatype.org","path":"/content/repositories/releases/org/mock-server/mockserver-netty/5.5.1/mockserver-netty-5.5.1-jar-with-dependencies.jar","port":443,"followAllRedirects":true} failed with HTTP status code 302
Warning: Task "download_jar" failed. Use --force to continue.
```

Am I missing something here?

**Comment by jamesdbloom** [comment]
That looks like a new bug however nothing I am aware of has changed in that code.  I'll try to look into it tomorrow.

**Comment by hennr** [comment]
Probably sonatype has added a redirect recently.
Your code does not follow redirects (302) but stops if http status code is > 300

**Comment by hennr** [comment]
Hi @jamesdbloom,

any chance to get an estimate for this? When it's fixed I can deal with #10
